lib/grp.gi

```diff
@@ -4398,55 +4398,48 @@ end);
 InstallMethod( GroupWithGenerators,
     "generic method for collection",
     [ IsCollection ],
-function( gens )
-local G,typ;
+    function( gens )
+    local typ;
 
-  typ:=MakeGroupyType(FamilyObj(gens),
+    gens:= AsList( gens );
+    typ:= MakeGroupyType( FamilyObj( gens ),
           IsGroup and IsAttributeStoringRep 
           and HasIsEmpty and HasGeneratorsOfMagmaWithInverses,
           gens,false,true);
 
-  G:=rec();
-  ObjectifyWithAttributes(G,typ,GeneratorsOfMagmaWithInverses,AsList(gens));
-
-  return G;
+    return ObjectifyWithAttributes( rec(), typ,
+               GeneratorsOfMagmaWithInverses, gens );
 end );
 
 InstallMethod( GroupWithGenerators,
     "generic method for collection and identity element",
-    IsCollsElms, [ IsCollection, IsMultiplicativeElementWithInverse ],
-function( gens, id )
-local G,typ;
+    IsCollsElms,
+    [ IsCollection, IsMultiplicativeElementWithInverse ],
+    function( gens, id )
+    local typ;
 
-  typ:=MakeGroupyType(FamilyObj(gens),
+    gens:= AsList( gens );
+    typ:= MakeGroupyType( FamilyObj( gens ),
           IsGroup and IsAttributeStoringRep 
             and HasIsEmpty and HasGeneratorsOfMagmaWithInverses and HasOne,
             gens,id,true);
 
-  G:=rec();
-  ObjectifyWithAttributes(G,typ,GeneratorsOfMagmaWithInverses,AsList(gens),
-                          One,id);
-
-  return G;
+    return ObjectifyWithAttributes( rec(), typ,
+               GeneratorsOfMagmaWithInverses, gens, One, id );
 end );
 
-InstallMethod( GroupWithGenerators,"method for empty list and element",
-  [ IsList and IsEmpty, IsMultiplicativeElementWithInverse ],
-  function( empty, id )
-local G,fam,typ;
-
-  fam:= CollectionsFamily( FamilyObj( id ) );
-
-  typ:=IsGroup and IsAttributeStoringRep
-        and HasGeneratorsOfMagmaWithInverses and HasOne and IsTrivial;
-  typ:=NewType(fam,typ);
+InstallMethod( GroupWithGenerators,
+    "method for empty list and element",
+    [ IsList and IsEmpty, IsMultiplicativeElementWithInverse ],
+    function( empty, id )
+    local typ;
 
-  G:= rec();
-  ObjectifyWithAttributes( G, typ,
-                            GeneratorsOfMagmaWithInverses, empty,
-                            One, id );
+    typ:= NewType( CollectionsFamily( FamilyObj( id ) ),
+              IsGroup and IsAttributeStoringRep and
+              HasGeneratorsOfMagmaWithInverses and HasOne and IsTrivial );
 
-  return G;
+    return ObjectifyWithAttributes( rec(), typ,
+               GeneratorsOfMagmaWithInverses, empty, One, id );
 end );
```

Comment:
I am not even sure what has changed in this function. Was anything changed, other than rearranging the code and reformatting it?